web/app/scripts/views/map/map-controller.js

```diff
@@ -88,6 +90,11 @@
                 params.polygon = userDrawnPoly;
             }
 
+            // Get a tilekey for the CSV export
+            QueryBuilder.djangoQuery(true, 0, _.extend({ limit: 0 }, params)).then(function(records) {
```

Comment:
Yeah, the endpoint is super fast, so it's not really a load concern. There is a potential concern around filling up the redis cache; under high enough load it might be possible for a user's tilekey to get pushed out of the cache before they have a chance to use it. So it's probably good not to treat tilekey requests as completely free, but they're certainly cheap.